README.rst

```diff
@@ -447,20 +447,21 @@ Get request accepts 3 params:
 
 
 
-For example if you are using axios to retrieve the comment list of second object (id=2) of a model (content type) called post.
+For example if you are using axios to retrieve the comment list of second object (id=2) of a model (content type) called post inside the app(django app) post.
 you can do the following:
 
 ::
 
-    $ curl -H "Content-Type: application/json" 'http://localhost:8000/api/comments/?type=MODEL_NAME&id=ID'
+    $ curl -H "Content-Type: application/json" 'http://localhost:8000/api/comments/?model_name=MODEL_NAME&model_id=ID&app_name=APP_NAME'
 
 
 **2- Create a comment or reply to an existing comment:**
 
 Authorization must be provided as a TOKEN or USERNAME:PASSWORD.
 
-- ``type``: is the model name of the content type that have comments associated with it.
-- ``id``: is the id of an object of that model
+- ``model_name``: is the model name of the content type that have comments associated with it.
+- ``app_name``: the name of the app that contains that model.
+- ``model_id``: is the id of an object of that model
 - ``parent_id``: is 0 or **NOT PROVIDED** for parent comments and for reply comments must be the id of parent comment
 
 
```

Comment:
The section below needs to be updated.

Reply:
is this not correct in its current state?

Reply:
It's correct but there might be few lines need to be updated for now I can see this example:
```bash
$ curl -X POST -u USERNAME:PASSWORD -d "content=CONTENT" -H "Content-Type: application/json" "http://localhost:8000/api/comments/create/?type=MODEL_NAME&id=ID&parent_id=0"
```

Reply:
yes, you're right. i have updated wherever i found the changes need to be made.